lib/rules/check-require.js

```diff
@@ -67,7 +82,7 @@ module.exports = {
     function visit(node) {
 
       function check(name) {
-        if (name[0] === '.' || name[0] === '/') {
+        if (name[0] === '.' || name[0] === '/' || UNSAFE_MODULE.test(name)) {
```

Comment:
is this additional condition needed? doesn't this plugin enforce relative paths like `./path/to/foo` over `path/to/foo`? I may not be understanding the intent here.

Reply:
I was trying to capture the entire path for internals like `internal/debugger/inspect`. But probably not necessary